### PR TITLE
Improving reporting unit search

### PR DESCRIPTION
**Pull request by creativeappdevamit**
# What and why?

# How to test?

# Trello
https://trello.com/c/XjLiwqc8/1004-spike-response-operations-performance-reporting-unit-page-1-2-days

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -8,13 +8,15 @@ build-kubernetes:
 	docker build -f _infra/docker/Dockerfile .
 
 lint:
-	pipenv check ./response_operations_ui ./tests
+	# commenting out pipenv check due to wtform issue. Once fixed we will bring it back
```
Comment (mfcrocker) [inline Makefile:11]: Have you put a card in to fix this?

File: _infra/helm/response-operations-ui/Chart.yaml [hunk 2]
```diff
@@ -14,8 +14,8 @@ type: application
 
 # This is the chart version. This version number should be incremented each time you make changes
 # to the chart and its templates, including the app version.
-version: 2.5.41
+version: 2.5.42
```
Comment (mfcrocker) [inline _infra/helm/response-operations-ui/Chart.yaml:17]:
```suggestion
version: 2.6.0
```

This is big enough for this

File: _infra/helm/response-operations-ui/Chart.yaml [hunk 3]
```diff
@@ -14,8 +14,8 @@ type: application
...
 
 # This is the version number of the application being deployed. This version number should be
 # incremented each time you make changes to the application.
-appVersion: 2.5.41
+appVersion: 2.5.42
```
Comment (mfcrocker) [inline _infra/helm/response-operations-ui/Chart.yaml:21]:
```suggestion
appVersion: 2.6.0
```

**Review by mfcrocker — APPROVED** [review]

File: Makefile (hunk 1 above)
Comment (creativeappdevamit) [reply Makefile:11]: yes - we are going to play it in our current sprint

File: _infra/helm/response-operations-ui/Chart.yaml (hunk 2 above)
Comment (creativeappdevamit) [reply _infra/helm/response-operations-ui/Chart.yaml:17]: 👍

File: _infra/helm/response-operations-ui/Chart.yaml (hunk 3 above)
Comment (creativeappdevamit) [reply _infra/helm/response-operations-ui/Chart.yaml:21]: 👍

**Pull request merged**